packages/editor-ui/src/components/WorkflowCard.vue

```diff
@@ -169,15 +168,8 @@ export default defineComponent({
 		},
 	},
 	methods: {
-		async onClick(event: Event) {
-			if (
-				this.$refs.cardActions === event.target ||
-				this.$refs.cardActions?.contains(event.target)
-			) {
-				return;
-			}
-
-			if (event.metaKey || event.ctrlKey) {
+		async onClick(event?: KeyboardEvent | PointerEvent) {
+			if (event?.metaKey || event?.ctrlKey) {
```

Comment:
Sure, done

Reply:
This change has broken the functionality and also the unit test.

With nullish coalescing if `event.metaKey` or `event.ctrlKey` is false, it will not cause a fallback to the other property which we need indeed

Rolling back the change